workflow-report/action.yml

```diff
@@ -0,0 +1,77 @@
+name: 'Sends VK Teams notifications about failed job'
```

Comment:
It should be identical with README.md, i.e `Report failed job to VK Team chat`.

Reply:
Done